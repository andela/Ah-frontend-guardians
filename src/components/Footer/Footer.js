import React , { Component } from 'react';

const Footer = () => {
        return (
            <div className="">
                <footer className="footer">
                <center>
                <p>Author's Haven .Inc</p>
                </center>
                </footer>
            </div>
        )
};

export default Footer;
