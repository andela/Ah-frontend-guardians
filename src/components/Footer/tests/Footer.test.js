import Footer from '../Footer';
import React from 'react';
import { shallow } from 'enzyme';

describe('Test the footer', () => {
    it('should render without crashing', () => {
        let wrapper = shallow(<Footer/>)
        expect(wrapper).toMatchSnapshot()
    })
})