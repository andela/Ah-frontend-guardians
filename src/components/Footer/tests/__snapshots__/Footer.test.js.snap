// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test the footer should render without crashing 1`] = `ShallowWrapper {}`;
